spec/features/user_can_log_out_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+# As a user
+# So that I can use the application securely
+# I can log out of my account and receive a confirmation
+
+RSpec.feature "Log out", type: :feature do
+  scenario "User logs in and immediately logs out with confirmation" do
+     sign_up
+     expect(page).to have_content("You have successfully logged in user: user@gmail.com")
```

Comment:
should this test include logging out and the content be 'you have successfully logged out?'